repos/system_upgrade/el7toel8/actors/dhclientreport/actor.py

```diff
@@ -0,0 +1,33 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+class CheckDhclientActor(Actor):
+    """
+    Check if dhclient is installed and report backward incompatible change in CLI options
+    """
+
+    name = "dhclient"
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+
+    def is_dhclient_installed():
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                if rpm.name == 'dhclient':
+                    return True
+        return False
+    
+    def process(self):
+        if is_dhclient_installed():
+            title = 'dhclient command line options changed'
+            summary = 'dhclient command line options have been changed ' \
+                      'since RHEL 7.7 release.'
+            remediation = 'If you have custom script which '\
+                          'involves dhclient please check dhclient(8) man page for ' \
+                          '-I/-C options'
+            severity = 'low'
+            report_with_remediation(title = title, severity = severity, remediation = remediation, summary = summary)
```

Comment:
I think this would trigger E251 Unexpected spaces around keyword

Please remove spaces around keyword params of report_with_remediation

Reply:
Done. Pep8 is happy now